src/amp-story-player-manager.js

```diff
@@ -99,8 +98,7 @@ export class AmpStoryPlayerManager {
     const doc = this.win_.document;
     const players = doc.getElementsByTagName('amp-story-player');
     for (let i = 0; i < players.length; i++) {
-      const playerEl = players[i];
-      const player = new AmpStoryPlayer(this.win_, playerEl);
+      const player = players[i];
       player.buildCallback();
```

Comment:
The custom element may not have been upgraded yet. You need to wait for https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry/whenDefined, which will resolve after all elements on the page are upgraded.